src/main/java/Parser/Parser.java

```diff
@@ -0,0 +1,86 @@
+package Parser;
```

Comment:
This file has incorrect indentation level, and the end of the file should have one newline character for Git